android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -24,6 +24,10 @@
   JNIEXPORT RETURN_TYPE Java_org_aomedia_avif_android_AvifDecoder_##NAME( \
       JNIEnv* env, jobject thiz, ##__VA_ARGS__)
 
+#define IGNORE_UNUSED_JNIPARAMETERS \
```

Comment:
nit: `IGNORE_UNUSED_JNI_PARAMETERS` (JNI and PARAMETERS are to be separated with _).